app/test_scan_360.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+from unittest.mock import MagicMock
+
+import numpy as np
+
+from app.scan360 import Scan360
+
+class TestScan360(unittest.TestCase):
+    def test_combine_scans(self):
+        bus = MagicMock()
+        c = Scan360(bus=bus.handle(), config={})
+        c.scan_front = np.ones(811, dtype=np.int32)
+        c.scan_rear = np.ones(811, dtype=np.int32)
+        c.joint_angle = 0
+        scan_360 = c.combine_scans()
```

Comment:
the tests show that would be maybe easier to understand if the parameters would be passed to the function (instead of using self.*)